plugins/com.google.cloud.tools.eclipse.sdk/META-INF/MANIFEST.MF

```diff
@@ -9,17 +9,21 @@ Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-ActivationPolicy: lazy
 Bundle-Localization: plugin
 Require-Bundle: com.google.cloud.tools.appengine;bundle-version="0.5.0"
-Import-Package: com.google.cloud.tools.eclipse.util.jobs,
+Import-Package: com.google.cloud.tools.eclipse.ui.util,
```

Comment:
Sorry I didn't get to reviewing this until now.  The other real comment is that we had the SDK stuff separated into a .sdk and .sdk.ui bundles.  This change has now pulled in UI stuff into both bundles.  I'm not sure this separation bought us much, so it might be worth merging the two bundles.